core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.ts

```diff
@@ -224,6 +245,26 @@ export class TemplateBuilderComponent implements OnInit, AfterViewInit, OnDestro
         this.store.removeRow(id as string);
     }
 
+    /**
+     * @description This method is used to update the layout properties
+     *
+     * @param {DotTemplateLayoutProperties} layoutProperties
+     * @memberof TemplateBuilderComponent
+     */
+    layoutPropertiesChange(layoutProperties: DotTemplateLayoutProperties): void {
+        this.store.updateLayoutProperties(layoutProperties);
+    }
+
+    /**
+     * @description This method is used to update the sidebar width
+     *
+     * @param {string} width
+     * @memberof TemplateBuilderComponent
+     */
+    sidebarWidthChange(width: string): void {
+        this.store.updateSidebarWidth(width);
+    }
```

Comment:
We do all this wrapping but the childrens of this component have access to the same instance of the store, so we can even call it from there.

Reply:
I'll evaluate this approach, looks promising